### PR TITLE
Feature/381 bug zero moving casecount

### PR DESCRIPTION
**Pull request by pendlrobinson**
Link to Jira Ticket: https://mathematicampr.atlassian.net/browse/C19ANDME-381

### CODE REVIEW

File: app/functions.R [hunk 1]
```diff
@@ -118,10 +119,21 @@ renderScoreHtml <- function(risk) {
...
         " to make sure you're well prepared in the days to come.")
-    )
-  )))
+    ))))
+  
+  if (moving_casecount == 0){
+    # if there are no reported cases in last 14 days
+    warning_text <- tags$p(style="color:#DF691A", 
+                           "WARNING: There are zero reported cases in the last 14 days. This could be due to USAFacts no longer reporting recent covid cases or deaths fr the county. 
+                           For more information, please check the ", tags$a("USAFacts", href = urls$usafacts_data), " or state agency's health department website.",
```
Comment (xindyhu) [inline app/functions.R:128]: A minor typo. "fr the county" should be "for the county".

**Review by xindyhu — CHANGES_REQUESTED** [review]

File: app/functions.R (hunk 1 above)
Comment (pendlrobinson) [reply app/functions.R:128]: Thank you for catching the typo. I corrected the text.

**Review by xindyhu — APPROVED** [review]

**Comment by xindyhu** [comment]
@pendlrobinson  Do you think this is worth reflecting in the changelog?

**Comment by pendlrobinson** [comment]
> @pendlrobinson Do you think this is worth reflecting in the changelog?

@xindyhu , good idea. I created a PR to document this update to the changelog.  Link to PR: https://github.com/mathematica-mpr/math-api-covid-risk-calculator/pull/201

**Comment by xindyhu** [comment]
> > @pendlrobinson Do you think this is worth reflecting in the changelog?
> 
> @xindyhu , good idea. I created a PR to document this update to the changelog. Link to PR: [mathematica-mpr/math-api-covid-risk-calculator#201](https://github.com/mathematica-mpr/math-api-covid-risk-calculator/pull/201)

Thank you @pendlrobinson , the PR looks good!

**Pull request merged**